hypha/settings/django.py

```diff
@@ -18,6 +18,7 @@
     "hypha.apply.review",
     "hypha.apply.determinations",
     "hypha.apply.stream_forms",
+    "hypha.apply.tasks",
```

Comment:
Please rename it to `user_tasks` or simply `todo` app. `tasks` too generic

Reply:
Or `user_actions`

Reply:
Renamed it to `todo`. `todo` makes more sense than others.